src/integrationTest/java/uk/gov/hmcts/reform/preapi/entities/AppAccessTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.preapi.entities;
+
+import jakarta.persistence.EntityManager;
+import jakarta.transaction.Transactional;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.preapi.Application;
+import uk.gov.hmcts.reform.preapi.enums.CourtType;
+
+import java.sql.Date;
+import java.sql.Timestamp;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(classes = Application.class)
+class AppAccessTest {
+
+    @Autowired
+    private EntityManager entityManager;
+
+    @Test
+    @Transactional
+    public void testSaveAndRetrieveAppAccess() { //NOPMD - suppressed JUnit5TestShouldBePackagePrivate
+        User user = HelperFactory.createUser(
```

Comment:
May want to consider a helper factory method to return a defaultTestUser that has all of these values prepopulated as it's used quite a lot in the test code?